module/proto/gravity/v1/attestation.proto

```diff
@@ -84,10 +84,20 @@ message EventSendToCosmosLocal {
   string amount = 4;
 }
 
```

Comment:
Now that we have Pending IBC Auto Forwards, I changed the events to clearly label them as Pending and added one for Executed forwards